docker/docker-compose-local.yml

```diff
@@ -22,6 +23,7 @@ services:
       - "30344:30344"
       - "9935:9935"
       - "9945:9945"
+      - "9615:9615"
     image: chevdor/polkadot:latest
```

Comment:
would be great to refer to the latest official release here

Reply:
fixed